internal/components/exporters_test.go

```diff
@@ -171,11 +192,215 @@ func TestDefaultExporters(t *testing.T) {
+		{
+			exporter: "honeycomb",
+			getConfigFn: func() config.Exporter {
+				cfg := expFactories["honeycomb"].CreateDefaultConfig().(*honeycombexporter.Config)
+				cfg.APIURL = "http://" + endpoint
+				cfg.APIKey = "busybeesworking"
+				return cfg
+			},
+		},
+		{
+			exporter: "humio",
+			getConfigFn: func() config.Exporter {
+				cfg := expFactories["humio"].CreateDefaultConfig().(*humioexporter.Config)
+				cfg.Endpoint = "http://" + endpoint
+				return cfg
+			},
+		},
+		{
+			exporter: "influxdb",
+			getConfigFn: func() config.Exporter {
+				cfg := expFactories["influxdb"].CreateDefaultConfig().(*influxdbexporter.Config)
+				cfg.Endpoint = "http://" + endpoint
+				return cfg
+			},
+		},
+		{
+			exporter: "loadbalancing",
+			getConfigFn: func() config.Exporter {
+				cfg := expFactories["loadbalancing"].CreateDefaultConfig().(*loadbalancingexporter.Config)
+				return cfg
+			},
+		},
+		{
+			exporter: "logzio",
+			getConfigFn: func() config.Exporter {
+				cfg := expFactories["logzio"].CreateDefaultConfig().(*logzioexporter.Config)
+				cfg.CustomEndpoint = "http://" + endpoint
+				return cfg
+			},
+		},
+		{
+			exporter: "loki",
+			getConfigFn: func() config.Exporter {
+				cfg := expFactories["loki"].CreateDefaultConfig().(*lokiexporter.Config)
+				cfg.Endpoint = "http://" + endpoint
+				return cfg
+			},
+		},
+		{
+			exporter: "newrelic",
+			getConfigFn: func() config.Exporter {
+				cfg := expFactories["newrelic"].CreateDefaultConfig().(*newrelicexporter.Config)
+				cfg.CommonConfig.HostOverride = "http://" + endpoint
+				return cfg
+			},
+		},
+		{
+			exporter: "sentry",
+			getConfigFn: func() config.Exporter {
+				cfg := expFactories["sentry"].CreateDefaultConfig().(*sentryexporter.Config)
+				return cfg
+			},
+		},
+		{
+			exporter: "skywalking",
+			getConfigFn: func() config.Exporter {
+				cfg := expFactories["skywalking"].CreateDefaultConfig().(*skywalkingexporter.Config)
+				return cfg
+			},
+		},
+		{
+			exporter:      "stackdriver",
+			skipLifecycle: true, // Is a deprecated exporter that has moved to google cloud exporter
+		},
+		{
+			exporter: "sumologic",
+			getConfigFn: func() config.Exporter {
+				cfg := expFactories["sumologic"].CreateDefaultConfig().(*sumologicexporter.Config)
+				cfg.Endpoint = "http://" + endpoint
+
+				return cfg
+			},
+		},
+		{
+			exporter: "tanzuobservability",
+			getConfigFn: func() config.Exporter {
+				cfg := expFactories["tanzuobservability"].CreateDefaultConfig().(*tanzuobservabilityexporter.Config)
+				cfg.Traces.Endpoint = "http://" + endpoint
+				return cfg
+			},
+		},
+		{
+			exporter: "tencentcloud_logservice",
+			getConfigFn: func() config.Exporter {
+				cfg := expFactories["tencentcloud_logservice"].CreateDefaultConfig().(*tencentcloudlogserviceexporter.Config)
+
+				return cfg
+			},
+		},
 	}
 
-	assert.Equal(t, len(tests)+25 /* not tested */, len(expFactories))
+	assert.Equal(t, len(tests)+0 /* not tested */, len(expFactories))
```

Comment:
I had left the untested number there to show that nothing is untested, but thinking about it, I should ideally remove this so that it is obvious that new components are to be added to the exporter life cycle test.

Reply:
Let me update this